src/components/RecipesDisplay/RecipesDisplay.css

```diff
@@ -0,0 +1,15 @@
+.RecipesDisplay {
```

Comment:
you should keep consistant naming with your css classes. I would stick with kebab case
```suggestion
.recipes-display {
```